src/vm.sh

```diff
@@ -51,11 +51,50 @@ function vm_up
         ${configurations[qemu_path_image]}
 }
 
+# This function manages ssh operations. Currently, we can have three basic
+# actions: entering in the target machine, execute the command in the VM, and
+# perform a local script inside the VM.
+#
+# @opts Expects a parameter to be passed for the ssh command. If this parameter
+#       receives a null value this function will make a simple ssh operation;
+#       otherwise, a command or script will be attempted to execute.
 function vm_ssh
 {
-  say "SSH"
-  say "-> Port: " ${configurations[ssh_port]} " IP: " ${configurations[ssh_ip]}
-  ssh -p ${configurations[ssh_port]} ${configurations[ssh_ip]}
+  local opts=$@
+  local port=${configurations[ssh_port]}
+  local target=${configurations[ssh_ip]}
+
+  # Mandatory parameter
+  if [ -z "$target" ]; then
+    complain "Invalid argument: $@"
+    exit 22 # EINVAL
+  fi
+
+  if [[ $# -gt 0 ]]; then
+    if [[ "$opts" =~ ^(--command|-c)= ]]; then
+      opts="$(echo $opts | cut -d = -f2 )"
+    elif [[ "$opts" =~ ^(--script|-s)= ]]; then
+      local script_path=$(echo $opts | cut -d = -f2 )
+
+      if [[ ! -f $script_path ]]; then
```

Comment:
Running something like `kw --script=~/kworkflow/tests/samples/dmesg` will failure this test, but I really don't understand why... I know that bash won't expand `~` for quoted strings such as `"~/"`, but we are not adding `"` here...